frontend/javascripts/oxalis/view/right-border-tabs/tree_hierarchy_view.js

```diff
@@ -260,37 +258,6 @@ class TreeHierarchyView extends React.PureComponent<Props, State> {
     this.props.onDeleteGroup(groupId);
   }
 
-  handleDropdownClick = (params: { domEvent: SyntheticMouseEvent<*>, key: string }) => {
-    const { domEvent, key } = params;
-    // $FlowIssue[prop-missing] .dataset is unknown to flow
-    const groupId = parseInt(domEvent.target.dataset.groupId, 10);
-    if (key === "create") {
-      this.createGroup(groupId);
-    } else if (key === "delete") {
-      this.deleteGroup(groupId);
-    } else if (key === "collapseSubgroups") {
-      this.setExpansionOfAllSubgroupsTo(groupId, false);
-    } else if (key === "expandSubgroups") {
-      this.setExpansionOfAllSubgroupsTo(groupId, true);
-    }
-  };
-
-  handleTreeDropdownMenuVisibility = (treeId: number, isVisible: boolean) => {
```

Comment:
It looks like one does not have to explicitly track the visibility state of the context menu anymore. Rather, antd seems to do that automatically/internally. Therefore, I removed this state handling.

Reply:
I just tested this PR and noticed that the "choose color" menu item for trees doesn't work properly anymore. The context menu closes immediately which makes the color picker pop up at x=0, y=0. I think, this is why the visibility was handled explicitly. Could you restore the state property and add a comment which explains why it is necessary?

Reply:
Good catch! I will revert this change.